F1WM/Repositories/NewsRepository.cs

```diff
@@ -79,6 +77,65 @@ public async Task<NewsDetails> GetNewsDetails(int id)
 			return news;
 		}
 
+		public async Task<NewsSummaryPaged> GetNewsByTagId(int? tagId, int page, int countPerPage)
+		{
+			await SetDbEncoding();
+			IQueryable<News> dbNews;
+
+			dbNews = context.NewsTagMatch
+					.Where(t => t.TagId == tagId)
+					.Include(t => t.News)
+					.Select(t => t.News)
+					.Where(n => !n.NewsHidden)
+					.Include(n => n.Tag);
+
+			return GetPagedResult(dbNews, page, countPerPage);
+		}
+
+		public async Task<NewsSummaryPaged> GetNewsByTypeId(int? typeId, int page, int countPerPage)
+		{
+			await SetDbEncoding();
+			IQueryable<News> dbNews;
+
+			dbNews = context.News
```

Comment:
No need to split dbNews declaration and assignment. You can just use `var dbNews = ...`